File: genealogy-tree-API/GenealogyTree.Domain/DTO/User/UserPositionModel.cs
using System;

namespace GenealogyTree.Domain.DTO.User
{
    public class UserPositionModel : PositionModel
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime UpdatedOn { get; set; }
        public string Type{ get; set; }//current, live, birth

#nullable enable
        public ImageFile? ImageFile { get; set; }
    }
}

File: gene-tree/src/app/modules/genealogy/models/user-position.model.ts
import { ImageFile } from './../../shared/models/image-file';
import { PositionModel } from './position.model';

export class UserPositionModel extends PositionModel {
  firstName: string;
  lastName: string;
  updatedOn: Date;
  imageFile: ImageFile;
  type: string;
}
